metadata-ingestion/tests/unit/test_mcp_builder.py

```diff
@@ -13,18 +13,35 @@ def test_guid_generator():
 
 def test_guid_generator_with_empty_instance():
     key = builder.SchemaKey(
-        database="test", schema="Test", platform="mysql", instance=None
+        database="test",
```

Comment:
There are some `instance="PROD"` in line 7 and 42 that may be semantically misleading. I would suggest to update them accordingly too.

Reply:
Thanks I renamed it to make it clearer a bit